txpipe/twopoint_fourier.py

```diff
@@ -551,7 +555,7 @@ def compute_power_spectra(
 
         # We need the theory spectrum for this pair
         # TODO: when we have templates to deproject, use this.
-        theory = cl_theory[(i, j, k)]
+        theory = cl_theory[(i, j, k)] # this is not needed anymore. Should I remove it? 
```

Comment:
What was that for?

Reply:
I think it was needed before to do a cl guess but now it uses the workspace, so I removed it.